MvvmCross/Platforms/Ios/Binding/Views/MvxSimpleTableViewSource.cs

```diff
@@ -27,6 +27,7 @@ public MvxSimpleTableViewSource(UITableView tableView, string cellIdentifier)
             : base(tableView)
         {
             _cellIdentifier = new NSString(cellIdentifier);
+            tableView.RegisterNibForCellReuse(UINib.FromName(cellIdentifier, NSBundle.MainBundle), cellIdentifier);
```

Comment:
What happens if there is no NIB/XIB with that name?

Reply:
Then people need to use the other ctor